### PR TITLE
hackathons: Add landing page for Athens hackathon

### PR DESCRIPTION
**Pull request by razvand**
Hackathon will take place on Thursday and Friday, March 30-31, 2023, as an in-person event in the Multimedia Amphitheater at the National Technical University of Athens

### CODE REVIEW

File: content/en/community/hackathons/2023-03-athens/_index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+The hackathon will take place as an in-person event at the Multimedia Amphitheatre at the Zografos campus of NTUA.
+Address is TODO
+TODO detailed instructions on reaching the location of the hackathon
+
+Support information and discussions will take place on [Discord](http://bit.ly/UnikraftDiscord) on the `#hack-athens23` channel.
+
+### Registration
+
+To take part in the hackathon please fill this [registration form](https://forms.gle/a315sJrzRQV8rZdz8) by **Sunday, March 19, 2023, 11pm CET**.
+
```
Comment (razvand) [inline content/en/community/hackathons/2023-03-athens/_index.md:23]: Please bring your own laptop. It’s best if you have a native Linux install on your laptop. For any situation, we will have virtual machines ready, to be able to get hacking quickly and easily!

**Review by StefanJum — CHANGES_REQUESTED** [review]
Just few very quick comments.

File: content/en/community/hackathons/2023-03-athens/_index.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+#### Thursday, March 30, 2023
+
+| Time (CET)    | Session                                                              | Presenter          |
+| ------------- | -------------------------------------------------------------------- | ------------------ |
+| 10:00 - 10:15 | High-level presentation of unikernels and Unikraft                   |                    |
+| 10:15 - 11:15 | Introduction to Unikraft. Building and running standard applications |                    |
+| 11:15 - 11:30 | Break                                                                |                    |
+| 11:30 - 12:45 | Inside Unikraft: Building, configuring, using different libraries    |                    |
+| 12:45 - 14:00 | Lunch                                                                |                    |
+| 14:00 - 15:00 | Running (Complex) Application in Unikraft                            |                    |
```
Comment (RaduNichita) [inline content/en/community/hackathons/2023-03-athens/_index.md:49]:
```suggestion
| 14:00 - 15:00 | Running (complex) applications in Unikraft                            |                    |
```

File: content/en/community/hackathons/2023-03-athens/_index.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+As part of the Unikraft community [Răzvan Deaconescu](https://github.com/razvand/) (UPB) and [TODO](TODO) will be present onsite with other community members providing support online, on Discord.
+
+### Schedule
+
+#### Thursday, March 30, 2023
+
+| Time (CET)    | Session                                                              | Presenter          |
+| ------------- | -------------------------------------------------------------------- | ------------------ |
+| 10:00 - 10:15 | High-level presentation of unikernels and Unikraft                   |                    |
```
Comment (RaduNichita) [inline content/en/community/hackathons/2023-03-athens/_index.md:44]:
```suggestion
| 10:00 - 10:15 | Overview of unikernels and Unikraft                   |                    |
```

**Review by RaduNichita — CHANGES_REQUESTED** [review]
@razvand see my comments.

File: content/en/community/hackathons/2023-03-athens/_index.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+collapsible: true
+description: Information on the Athens Unikraft Hackathon, March 30-31, 2023
+image: /assets/imgs/2023-03-hack-athens.png
+---
+
+## Athens Unikraft Hackathon
+
+Unikraft with [Nubificus](https://nubificus.co.uk/), [the Institute of Communication and Computer Systems (ICCS)](https://www.iccs.gr/en/) and [the National Technical University of Athens (NTUA)](http://www.ntua.gr/index_en.html) come together to organize the **Unikraft Athens Hackathon** to be held on Thursday and Friday, March 30-31, 2023.
+
+The hackathon will take place as an in-person event at the Multimedia Amphitheatre at the Zografos campus of NTUA.
```
[the remark this answers is not in the document]
Comment (michpappas) [reply content/en/community/hackathons/2023-03-athens/_index.md:14]: Actually the area is called Zografou, so it should be "Zografou campus" here and elsewhere. http://old.ntua.gr/map_en.html.

File: content/en/community/hackathons/2023-03-athens/_index.md (hunk 2 above)
Comment (razvand) [reply content/en/community/hackathons/2023-03-athens/_index.md:49]: This is correct. If parentheses were missing, you would write with capital `Complex`.

File: content/en/community/hackathons/2023-03-athens/_index.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
+---
+title: Athens 2023
+date: 2023-01-20T07:20:00+02:00
+draft: false
+collapsible: true
+description: Information on the Athens Unikraft Hackathon, March 30-31, 2023
+image: /assets/imgs/2023-03-hack-athens.png
```
Comment (RaduNichita) [inline content/en/community/hackathons/2023-03-athens/_index.md:7]: Should the photo be added as well?

File: content/en/community/hackathons/2023-03-athens/_index.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+The hackathon will take place as an in-person event at the Multimedia Amphitheater at the Zografou campus of NTUA.
+The full address is Multimedia Amphitheater, Central Library Building, Heroon Polytechniou 9, 15780 Zografou, Greece
+TODO detailed instructions on reaching the location of the hackathon
+
+Support information and discussions will take place on [Discord](http://bit.ly/UnikraftDiscord) on the `#hack-athens23` channel.
+
+### Registration
+
+To take part in the hackathon please fill this [registration form](https://forms.gle/a315sJrzRQV8rZdz8) by **Sunday, March 19, 2023, 11pm CET**.
```
Comment (RaduNichita) [inline content/en/community/hackathons/2023-03-athens/_index.md:22]: The photo in the form is wrong. It shows details about the Munich Hackathon.

**Review by RaduNichita — CHANGES_REQUESTED** [review]
I left some other comments.

**Review by michpappas — CHANGES_REQUESTED** [review]
Image looks awesome 😎. Just a few minor comments.

File: content/en/community/hackathons/2023-03-athens/_index.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (razvand) [reply content/en/community/hackathons/2023-03-athens/_index.md:22]: fixed

File: content/en/community/hackathons/2023-03-athens/_index.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+date: 2023-01-20T07:20:00+02:00
+draft: false
+collapsible: true
+description: Information on the Athens Unikraft Hackathon, March 30-31, 2023
+image: /assets/imgs/2023-03-hack-athens.jpg
+---
+
+## Athens Unikraft Hackathon
+
+Unikraft, [Nubificus](https://nubificus.co.uk/), [the High Speed Communication Networks Lab (HSCN)](http://hscnl.ece.ntua.gr/) and [the Computing Systems Lab (CSLab)](http://www.cslab.ece.ntua.gr/) from the National Technical University of Athens ([ICCS](https://www.iccs.gr/en/)/[NTUA](http://www.ntua.gr/index_en.html)) come together to organize the **Unikraft Athens Hackathon** to be held on Thursday and Friday, March 30-31, 2023.
```
Comment (StefanJum) [inline content/en/community/hackathons/2023-03-athens/_index.md:12]: The [NTUA link](http://www.ntua.gr/index_en.html) gives a 404.

File: content/en/community/hackathons/2023-03-athens/_index.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+As part of the Unikraft community [Răzvan Deaconescu](https://github.com/razvand/) and [Ștefan Jumărea](https://github.com/StefanJum) will be present on-site, with other community members providing support online, on Discord.
+
+### Schedule
+
+#### Thursday, March 30, 2023
+
+| Time (CET)    | Session                                                              | Presenter          |
+| ------------- | -------------------------------------------------------------------- | ------------------ |
+| 10:00 - 10:15 | Overview of unikernels and Unikraft                                  |                    |
```
Comment (StefanJum) [inline content/en/community/hackathons/2023-03-athens/_index.md:47]:
```suggestion
| 10:00 - 10:15 | Overview of Unikernels and Unikraft                                  |                    |
```

File: content/en/community/hackathons/2023-03-athens/_index.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+| 10:15 - 11:15 | Introduction to Unikraft. Building and running standard applications |                    |
```
Comment (StefanJum) [inline content/en/community/hackathons/2023-03-athens/_index.md:48]:
```suggestion
| 10:15 - 11:15 | Introduction to Unikraft: Building and running standard applications |                    |
```

File: content/en/community/hackathons/2023-03-athens/_index.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+| 11:15 - 11:30 | Break                                                                |                    |
+| 11:30 - 12:45 | Unikraft Internals: Building, configuring, using different libraries |                    |
+| 12:45 - 14:00 | Lunch                                                                |                    |
+| 14:00 - 15:00 | Running (Complex) Application in Unikraft                            |                    |
+| 15:00 - 16:00 | Porting applications to Unikraft                                     |                    |
```
Comment (StefanJum) [inline content/en/community/hackathons/2023-03-athens/_index.md:53]:
```suggestion
| 15:00 - 16:00 | Porting Applications to Unikraft                                     |                    |
```

**Review by StefanJum — CHANGES_REQUESTED** [review]
Looks really good @razvand, just a few comments.

File: content/en/community/hackathons/2023-03-athens/_index.md (hunk 7 above)
Comment (michpappas) [reply content/en/community/hackathons/2023-03-athens/_index.md]: I think https://www.ntua.gr is good enough, as it seems to redirect to https://www.ntua.gr/en if you access from abroad.

**Review by michpappas — APPROVED** [review]
Looks great!

Reviewed-by: Michalis Pappas <mpappas@fastmail.fm>

**Review by StefanJum — APPROVED** [review]
All good now, thanks.

Approved-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Pull request merged**